src/v/cloud_storage/remote.cc

```diff
@@ -792,4 +793,42 @@ cloud_roles::credentials auth_refresh_bg_op::build_static_credentials() const {
       _region_name};
 }
 
+std::vector<s3::object_tag> remote::get_manifest_tags(
+  const model::ntp& ntp, model::initial_revision_id rev) {
+    auto tags = default_partition_manifest_tags;
+    tags.push_back({.key = "rp-ns", .value = ntp.ns()});
```

Comment:
Since there is a fixed number of tags, can we construct the vector with an explicit reserved size?